core/variant_call.cpp

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```

Comment:
Looks like your IDE/code editor has added a BOM (Byte Order Mark) here, could you remove it?